src/ubx.cpp

```diff
@@ -501,6 +501,10 @@ int GPSDriverUBX::configureDevicePreV27(const GNSSSystemsMask &gnssSystems)
 		return -1;
 	}
 
+	if (!configureMessageRateAndAck(UBX_MSG_RXM_RTCM, 1, true)) {
```

Comment:
Can this fail on devices w/o rtcm support?

Reply:
Maybe I should remove it for older devices.

Reply:
It looks like the M9N has the message even though it doesn't have RTCM. It looks like the M8 doesn't have it.